.travis.yml

```diff
@@ -20,17 +21,19 @@ matrix:
     - php: 5.6
       env: SYMFONY_VERSION=2.6.*
     - php: 5.6
-      env: SYMFONY_VERSION=2.7.*@dev
+      env: SYMFONY_VERSION=2.7.*
     - php: 5.6
-      env: SYMFONY_VERSION=2.8.*@dev
+      env: SYMFONY_VERSION=2.8.*
+    - php: 7.0
+      env: SYMFONY_VERSION=2.8.*
```

Comment:
there is no reason to run tests for Symfony 2.8 twice, on 5.6 and 7.0. The bundle is already tested on both different versions of PHP, and Symfony itself too.